bin/isolated-functions.sh

```diff
@@ -443,6 +443,13 @@ case "${NOCOLOR:-false}" in
 		;;
 esac
 
+if [[ -z "${NO_COLOR}" ]] ; then
+	__set_colors
+else
+	__unset_colors
+fi
```

Comment:
I think we don't want to undo a previous `NOCOLOR=true` with an unset or empty `NO_COLOR` here.

Reply:
Ah yes I'll refactor it accordingly and put nocolor inside the if block of no_color. So if no_color is empty or unset then case nocolor will happen with a default of false

Reply:
Does the new implementation work ?